meta/autoid/autoid.go

```diff
@@ -299,9 +306,12 @@ func (alloc *allocator) rebase4Sequence(tableID, requiredBase int64) error {
 	// TODO: sequence metrics
 	metrics.AutoIDHistogram.WithLabelValues(metrics.TableAutoIDRebase, metrics.RetLabel(err)).Observe(time.Since(startTime).Seconds())
 	if err != nil {
-		return err
+		return 0, false, err
 	}
-	return nil
+	if alreadySatisfied {
+		return 0, true, nil
+	}
+	return requiredBase, false, nil
```

Comment:
```suggestion
	return requiredBase, false, err
```
Then we can remove line308-310